geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStore.scala

```diff
@@ -78,17 +81,22 @@ class KafkaDataStore(
     KafkaDataStore.producer(config)
   }
 
+  // view type name -> actual type name
+  private val layerViewLookup =
+    config.layerViewsConfig.flatMap { case (typeName, views) => views.map(_.typeName -> typeName).toMap }
```

Comment:
Silly corner case:  Is there any checking that the configuration doesn't have a collision in the layernames?

Reply:
there is, here: https://github.com/elahrvivaz/geomesa/blob/fcr_kafka_layer_filters/geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreFactory.scala#L329-L332

Reply:
Awesome.